### DISCUSSION TITLE
Changes in repo we can do

### DISCUSSION THREAD

**Discussion by kibergrad** [cyberbotics/wrestling#80] [General]
@omichel what is the level of changes we can do in our repo apart from the controllers file.
For example, can we change or add some nodes?

**Comment by omichel** [comment]
You can modify or add any file inside your repository. However, the competition will use only your [Dockerfile](https://github.com/cyberbotics/wrestling/blob/main/controllers/Dockerfile) to launch your robot controller (by default in your [participant](https://github.com/cyberbotics/wrestling/tree/main/controllers/participant) folder). It will not use your possibly modified [world file](https://github.com/cyberbotics/wrestling/blob/main/worlds/wrestling.wbt), [referee](https://github.com/cyberbotics/wrestling/tree/main/controllers/referee) or [opponent](https://github.com/cyberbotics/wrestling/tree/main/controllers/opponent) folder. Instead, the world file and referee will be taken from the original repository and the opponent folder will be taken from your opponent's repository.
